src/App.tsx

```diff
@@ -166,6 +167,11 @@ class App extends RX.Component<{}, AppState> {
     RX.Modal.show(<AuthenticationDialog manager={this.authenticationManager} closeCallback={onClose} />, "AuthenticationDialog");
   }
 
+  // TODO: use a better method of logging out than simply refreshing the page
```

Comment:
I think it's okay to just refresh the page, for now.

Maybe later we can see if it can be useful to implement an automated login to known servers, then it would be necessary to remove the entry from localStorage before refreshing if the user checked the `Remember the certificate pair on this device/browser` box.